doc/man3/X509_STORE_CTX_get_by_subject.pod

```diff
@@ -28,12 +31,24 @@ stores the looked up object in I<ret>.
 X509_STORE_CTX_get_obj_by_subject() is like X509_STORE_CTX_get_by_subject()
 but returns the found object on success, else NULL.
 
+X509_STORE_CTX_get1_objs_by_auth() finds all certificates in the store that
+match the given subject I<name> and, if provided, the authority key ID I<akid>.
+If both I<name> and I<akid> are supplied, the function requires both values
+to match. The authority key ID is matched against the subject key ID of the
+certificates in the store. Certificates are matched in the order they
+are stored.
+
 =head1 RETURN VALUES
 
 X509_STORE_CTX_get_by_subject() returns 1 if the lookup was successful, else 0.
 
 X509_STORE_CTX_get_obj_by_subject() returns an object on success, else NULL.
 
+X509_STORE_CTX_get1_objs_by_auth() returns a new stack with pointers to all
+matching certificates in the store. The caller must free the returned stack.
+If no certificate matches the given subject, an empty stack is returned.
+In case of any error, NULL is returned.
+
```

Comment:
Please add a `HISTORY` section and mention this function there. It will go into 3.5 (not 3.4 as that is feature frozen now).